iron-menu-behavior.html

```diff
@@ -148,7 +148,12 @@
         var item = this.items[(curFocusIndex - i + length) % length];
         if (!item.hasAttribute('disabled')) {
           this._setFocusedItem(item);
-          return;
+
+          // Focus might not have worked, if the element was hidden or not
+          // focusable. In that case, try again.
+          if (document.activeElement == item) {
```

Comment:
I think you'll need to specifically check the item's owner root's activeElement, since [children can come from anywhere in light / shadow thanks to `queryDistributedElements`](https://github.com/PolymerElements/iron-menu-behavior/blob/5576a3e/iron-menu-behavior.html).